### DISCUSSION TITLE
Suggest adding WSD to Samba instructions

### DISCUSSION THREAD

**Discussion by KMoonraiser** [thinkst/opencanary#269] [Feature Requests]
It makes sense that we want our canary to be as visible as possible. Installing WSD makes finding it easier:

https://devanswers.co/discover-ubuntu-machines-samba-shares-windows-10-network/

**Comment by jayjb** [comment]
Hi @KMoonraiser,

Sorry for the delay! This sounds like an interesting approach. I'm going to have a discussion with the team first. I am redoing some of the instructions and slipping this in sounds useful.
